resources/dicts/events/death/general.json

```diff
@@ -4872,17 +5138,16 @@
         }
     },
     {
-        "event_id": "gen_death_",
+        "event_id": "gen_death_age_anylead1",
+        "biome": ["any"],
         "camp": [
             "any"
         ],
         "season": [
             "any"
         ],
-        "tags": [
-            "old_age",
-            "some_lives"
-        ],
+        "sub_type": [ "old_age" ],
+        "tags": [],
```

Comment:
add tag "some_lives" so that the leader will lose more than one life, but not all lives